licensescout-maven-utils/src/test/java/org/aposin/licensescout/maven/utils/ArtifactHelperTest.java

```diff
@@ -105,9 +119,39 @@ public RepositorySystem getRepositorySystem() {
                 return remoteRepositories;
             }
         };
+        return repositoryParameters;
+    }
 
-        final File resultFile = ArtifactHelper.getArtifactFile(repositoryParameters, artifactItem);
-        return resultFile;
+    /**
+     * Test case for the method {@link ArtifactHelper#getDependencies(IRepositoryParameters, List, ArtifactScope)}.
```

Comment:
Please introduce better names or better explanation of what is going tested here and what is the difference among the two tests. Thanks

Reply:
Reworked tests, also added new ones.